code/modules/mob/living/carbon/human/login.dm

```diff
@@ -1,3 +1,6 @@
 /mob/living/carbon/human/Login()
-	..()
-	if(species) species.handle_login_special(src)
+	. = ..()
+	species?.handle_login_special(src)
+	if(away_time)
+		deltimer(away_time)
+		away_time = 0
```

Comment:
```suggestion
		away_time = null
```

Reply:
Will need other references to be changed, since it's no longer a time reference, but a timer ID.

Reply:
The default value is 0, though I suppose this would be helpful for clarity that it triggered.